js/fileexplorer/Index.svelte

```diff
@@ -37,6 +38,11 @@
 	export let interactive: boolean;
 
 	$: rerender_key = [root_dir, glob, ignore_glob];
+
+	$: if (JSON.stringify(value) !== JSON.stringify(old_value)) {
```

Comment:
Let's add a unit test for this - otherwise looks good to me @aliabid94 !

Reply:
done